UPDATING.md

```diff
@@ -78,50 +78,76 @@ This section describes the changes that have been made, and what you need to do
 
 #### Simplification of CLI commands
 
-The ability to manipulate users from the command line has been changed. 'airflow create_user' and 'airflow delete_user' and 'airflow list_users' has been grouped to a single command `airflow users` with optional flags `--create`, `--list` and `--delete`.
+The ability to manipulate users from the command line has been changed. ``airflow create_user``,  ``airflow delete_user``
+ and ``airflow list_users`` has been grouped to a single command `airflow users` with optional flags `create`, `list` and `delete`.
```

Comment:
In my opinion, these sections can be combined to the next, but examples can be added below the table.